### ISSUE TITLE
Unknown reponse for CHLSTATUS inquiry

### ISSUE THREAD

**Issue by pkaczynski** [dsuch/pymqi#59]
`MQMIError` seems to have erraneous errStringDicts definition in line 1207. 
```
errStringDicts = (_MQConst2String(CMQC, "MQRC_"), _MQConst2String(CMQC, "MQRCCF_"),)
```
AFAIK the latter element of the tuple translates constants from `CMQC` module whose names starts with `MQRCCF`. However, there are no constants like this in this module. They are present in `CMQCFC`.

I think that is why a code like this:
```
pcf = pymqi.PCFExecute('SOME_VALID_QMGR')
response = pcf.MQCMD_INQUIRE_CHANNEL_STATUS({
    pymqi.CMQCFC.MQCACH_CHANNEL_NAME: 'nonexistent channel status'
})
```
Results in `MQMIError` with `WTF? Error code 3065 not defined`. Let me know if I'm correct. If you need a pull request, let me know.

**Comment by dsuch** [comment]
Apologies for the language used in the error message, it was introduced prior to my becoming the maintainer of this project.

Your reasoning sounds correct, do you think you can procure a PR for it, one that would take into account the presence of constants in all of CM*.py modules?

**Comment by dsuch** [comment]
Also, I wonder if #57 is not related?

**Comment by SeyfSV** [comment]
Fixed in 1aab754b0b100496465c89941f23803962e0bee0
